Lib/threading.py

```diff
@@ -34,6 +34,10 @@
 _allocate_lock = _thread.allocate_lock
 _set_sentinel = _thread._set_sentinel
 get_ident = _thread.get_ident
+try:
+    get_native_id = _thread.get_native_id
+except AttributeError:
+    get_native_id = lambda: None
```

Comment:
Hum no, I would prefer to not define the function if it is not supported. Returning None contradicts the doc.

Reply:
@vstinner Re: my comment above, when you say "not define the function" do you mean simply declare `get_native_id = None`, or remove it altogether from the module (via `del` or some similar mechanism)?